src/jit/compiler.h

```diff
@@ -960,12 +972,66 @@ struct CompTimeInfo
     unsigned __int64 m_parentPhaseEndSlop;
     bool             m_timerFailure;
 
+#if MEASURE_CLRAPI_CALLS
+    // The following measures the time spent inside each individual CLR API call.
+    unsigned         m_allClrAPIcalls;
+    unsigned         m_perClrAPIcalls [API_ICorJitInfo_Names::API_COUNT];
+
+    unsigned __int64 m_allClrAPIcycles;
+    unsigned __int64 m_perClrAPIcycles[API_ICorJitInfo_Names::API_COUNT];
+
+    unsigned __int32 m_maxClrAPIcycles[API_ICorJitInfo_Names::API_COUNT];
+#endif // MEASURE_CLRAPI_CALLS
```

Comment:
Actually, no - it's cycles inside the call which better fit in a 32-bit int. I'm trying to save memory because these arrays are not that small.